src/optimization.py

```diff
@@ -2,12 +2,12 @@
 import gurobipy as gp
 
 
-# bounds = ((min_f1, max_f1), ..., (min_fk, max_fk))
-# A f <= b
-# A is a matrix and b is a column vector
-# return (throughput_f1, ... throughput_fk) > (0, ..., 0) if feasible;
-# otherwise (0, ..., 0)
 def maximizeMarginalRate(bounds, A, b, noLog=True):
+    # bounds = ((min_f1, max_f1), ..., (min_fk, max_fk))
```

Comment:
Thanks. If it is possible, I'd suggestion adding the definition of A, b so that we know what is the input.